components/frontapp/actions/list-message-templates/list-message-templates.mjs

```diff
@@ -0,0 +1,61 @@
+import frontApp from "../../frontapp.app.mjs";
+
+export default {
+  key: "frontapp-list-message-templates",
+  name: "List Message Templates",
+  description: "List the message templates. [See the documentation](https://dev.frontapp.com/reference/list-message-templates).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    frontApp,
+    sortBy: {
+      type: "string",
+      label: "Sort By",
```

Comment:
Note: the docs do not mention a page or per_page parameter for this endpoint at all

Reply:
The docs show it uses a `page_token` instead. This action should have a `maxResults` prop and return a paginated list as the result. Note: You can look at how the `messageTemplateId` prop uses pagination.